src/cloudcasting/validation.py

```diff
@@ -436,3 +454,28 @@ def validate(
                 channel_ind=int(channel_ind),
                 fps=1,
             )
+
+
+def validate_from_config(
+    config_file: Annotated[str, typer.Option(help="Path to config file")] = "validate_config.yml",
+    model_file: Annotated[
+        str, typer.Option(help="Name of Python file with model definition (without .py)")
+    ] = "model",
+) -> None:
+    """CLI function to validate a model from a config file.
+
+    Args:
+        config_file: Path to config file
+        model_file: Name of Python file with model definition (without .py)
+    """
+    with open(config_file) as f:
+        config: dict[str, Any] = yaml.safe_load(f)
+
+    if model_file.strip()[-3:] == ".py":
+        model_file = model_file.split(".py")[0].strip()
+
+    module = importlib.import_module(model_file)
```

Comment:
Does this work if `model_file` is not in the current working directory?

Reply:
It does now, good suggestion!